admin/src/components/StatusCard.tsx

```diff
@@ -1,50 +1,110 @@
+import { useState, useEffect } from 'react';
 import { HiUsers } from 'react-icons/hi2';
 import { LiaNewspaperSolid } from 'react-icons/lia';
-import { TbActivityHeartbeat } from 'react-icons/tb';
 import { Link } from 'react-router-dom';
+import { tokenState } from '../store/atoms/auth';
+import { useRecoilValue } from 'recoil';
+import { MdAddReaction } from "react-icons/md";
+import { SiGooglemessages } from "react-icons/si";
+import { RiHeartsFill } from "react-icons/ri";
+import { FaComments } from "react-icons/fa6";
+import axios from 'axios';
 
 const StatusCard = () => {
+  const [stats, setStats] = useState({ totalUsers: 0, totalPosts: 0, totalComments: 0, totalReactions: 0, messages: 0, favorites: 0 });
```

Comment:
You can use a variable to save initial stats.

```suggestion
  const initialStats = { totalUsers: 0, totalPosts: 0, totalComments: 0, totalReactions: 0, messages: 0, favorites: 0 };
  const [stats, setStats] = useState(initialStats);
```

Reply:
Completed !